### PR TITLE
Create Account Popup

### PR DESCRIPTION
**Pull request by Scholasticpal**
fixes- CreateAccount Popup completed.
Please have a look.

### CODE REVIEW

**Review by HemanthSai7 — APPROVED** [review]
LGTM🚀!

File: frontend/homepage/src/components/Home_p1/Navbar/CreateAccount/CreateAccount.js [hunk 1]
```diff
@@ -34,29 +30,47 @@ function CreateAccount(props) {
...
           </label>
           <br />
+          {isOwner&& <div>
           <label>
             Role:
-            <select onChange={IsOwner} required name="role">
+            
+            <select required name="role">
               <option defaultValue="None">None</option>
               <option value="GOVTOff">Owner</option>
```
Comment (mayureshagashe2105) [inline frontend/homepage/src/components/Home_p1/Navbar/CreateAccount/CreateAccount.js:39]: Please comment out this line as we are not allowing the owner to create more owner accounts.

**Review by mayureshagashe2105 — CHANGES_REQUESTED** [review]
Please make the suggested changes and commit again. Thank you

File: frontend/homepage/src/components/Home_p1/Navbar/CreateAccount/CreateAccount.js (hunk 1 above)
Comment (Scholasticpal) [reply frontend/homepage/src/components/Home_p1/Navbar/CreateAccount/CreateAccount.js]: the changes are now made, have a look.

**Review by HemanthSai7 — APPROVED** [review]
LGTM🚀

File: frontend/homepage/src/components/Home_p1/Navbar/CreateAccount/CreateAccount.js (hunk 1 above)
Comment (mayureshagashe2105) [reply frontend/homepage/src/components/Home_p1/Navbar/CreateAccount/CreateAccount.js]: Looks good, merging!

**Review by mayureshagashe2105 — APPROVED** [review]
LGTM!

**Pull request merged**